src/code.h

```diff
@@ -26,10 +26,6 @@
 **  Header for any statement or expression encoded in a function body.
 */
 typedef struct {
-    // `visited` starts out as 0 and is set to 1 if the statement or
-    // expression has ever been executed while profiling is turned on
-    unsigned int visited : 1;
-
     // `line` records the line number in the source file in which the
     // statement or expression started
     unsigned int line : 31;
```

Comment:
Might as well put that bit to use again (and possibly allow generation of slightly nicer code?)
```suggestion
    unsigned int line : 32;
```